packages/cli/generators/relation/relationBelongsTo.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
```

Comment:
I am proposing to use a slightly different file name: `belongs-to.generator.js` or `belongs-to-relation.generator.js`.

What's important: use kebab-case (not camelCase), add suffix based on the file type (`.generator.js`).

The same comment applies to other generator files too.

Reply:
Fixed.